website/content/tools/autoscaling/plugins/apm/datadog.mdx

```diff
@@ -18,6 +18,7 @@ apm "datadog" {
   config = {
     dd_api_key = "<api key>"
     dd_app_key = "<app key>"
+    dd_api_rate_per_second = "<200>"
```

Comment:
The `< >` are here for the API key to signify that we're asking the user to fill in a long/secret string.

```suggestion
    dd_api_rate_per_second = "200"
```

(And I guess this can't be `200` instead of "200" because of the way autoscaler parses its plugin configs into a `map[string]string`?)